Shared/Supporting Files/Views/SampleInfoView.swift

```diff
@@ -33,7 +33,7 @@ struct SampleInfoView: View {
             WebView(htmlString: codeHTML)
                 .opacity(informationMode == .code ? 1 : 0)
         }
-        .edgesIgnoringSafeArea([.horizontal, .bottom])
+        .ignoresSafeArea(edges: [.horizontal, .bottom])
```

Comment:
Is this modifier actually needed? I can't see a difference without it...

Reply:
Good question. It makes a tiny difference with/without it. 

Without bottom edge ignore, the README won't overlap with the screen bottom home screen drag indicator.
Without horizontal ignore, when zoom the webview/scroll view in landscape orientation, the content won't extend into left and right.